sim/core/buffs.go

```diff
@@ -197,7 +199,9 @@ func applyBuffEffects(agent Agent, raidBuffs proto.RaidBuffs, partyBuffs proto.P
 		})
 	}
 	if raidBuffs.WrathOfAirTotem {
-		character.PseudoStats.CastSpeedMultiplier *= 1.05
+		character.AddStats(stats.Stats{
+			stats.SpellHaste: 5.0*HasteRatingPerHastePercent,
```

Comment:
This is a DPS change for all classes, I think it's incorrect unless you have some data backing it up.